symbiflow.py

```diff
@@ -321,3 +329,158 @@ def check_env():
             'yosys': have_exec('yosys'),
             'vpr': have_exec(VPR.vpr_bin()),
         }
+
+
+class Nextpnr(Toolchain):
```

Comment:
Should this be NextpnrXilinx or something? It wouldn't work for iCE40 or ECP5 from what I can see here?